sdk-setup/src/sdk-assistant

```diff
@@ -210,7 +217,7 @@ case ${1:-} in
         ;;
 esac
 
-[[ $# -gt 0 ]] || fail "command expected"
+[[ $# -gt 0 ]] || usage "Command expected"
```

Comment:
In this case the help text is still not written to stderr while it should be. Better would be dropping the "exit with error if any argument is given" functionality from usage() and let the caller side care about:

`[[ $# -gt 0 ]] || { usage >&2; fail "Command expected; }`

Reply:
Ok, I can do that as well. I thought you specifically wanted only the error messages go to stderr. The help text is not really error message.

Reply:
except that doing it the way you suggest would be silly as it would print "Try 'sdk-assistant --help' for usage information" right below the usage information...